### PR TITLE
Embedded hal 1.0.0 alpha.3

### PR DESCRIPTION
**Pull request by eldruin**
Builds on and superseeds #43 and #48.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 - Set default features to build both sysfs and cdev pin types
 - Removed `Pin` export, use `CdevPin` or `SysfsPin`
+- Adapted to `embedded-hal` `1.0.0-alpha.2` release.
```
Comment (ryankurte) [inline CHANGELOG.md:14]: .3?

**Review by ryankurte — DISMISSED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (eldruin) [reply CHANGELOG.md]: Good catch. It should be fixed now.

**Review by ryankurte — APPROVED** [review]
lgtm, lets get started on the new series then eh ^_^

bors r+

**Comment by eldruin** [comment]
bors retry

**Comment by posborne** [comment]
bors retry

**Pull request merged**